app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorViewTest.kt

```diff
@@ -133,6 +136,7 @@ class CreditCardEditorViewTest {
         val cardNumber = "4111111111111110"
         val expiryMonth = 5
         val expiryYear = calendar.get(Calendar.YEAR)
+        expiryYear.toString()
```

Comment:
Do we need this?

Reply:
Don't see a response to this, but also marked as resolved. I am assuming we should remove this?